src/Module/GetEddServicePricesCapableTrait.php

```diff
@@ -55,10 +56,14 @@ protected function _getEddServicePrices($serviceId)
         $index   = 0;
 
         foreach ($lengths as $_lengthInfo) {
+            $_length = (int) $this->_containerGet($_lengthInfo, 'sessionLength');
+            $_name   = CarbonInterval::seconds($_length)->cascade()->forHumans();
```

Comment:
I wonder from where Carbon knows how to localize the human-readable length.

Reply:
It uses the PHP locale, according to its [documentation](https://carbon.nesbot.com/docs/#api-localization).

Reply:
I must say that I dislike the static part very, very much.

Reply:
Same. But it beats the alternative right; implementing our own 😆 

FYI: I looked at other date time packages, such as Brick. But they all seem to enjoy using static calling. Crazy!

Reply:
@mecha, Carbon docs appear to indicate that it's possible to work with instances too. But if I understood correctly, they are instances of actual datetimes, not of the library which has the functionality to deal with them.